packages/flutter/lib/src/rendering/object.dart

```diff
@@ -3317,12 +3317,16 @@ abstract class RenderObject with DiagnosticableTreeMixin implements HitTestTarge
   }
 
   /// {@template flutter.rendering.RenderObject.getTransformTo}
```

Comment:
What about the other instances where this doc macro is used? Do all of these changes also apply there?

Reply:
Moved the `paintsChild` paragraph up so it's included in the template.